docs/sources/administration/home_dashboard.md

```diff
@@ -0,0 +1,36 @@
++++
+title = "How to replace the default home dashboard"
+description = "How to replace the default home dashboard"
+keywords = ["grafana", "configuration", "documentation", "home"]
+type = "docs"
+[menu.docs]
+name = "Home Dashboard"
+identifier = "home-dashboard"
+parent = "admin"
+weight = 100
++++
+
+# How to replace the default home dashboard
+
+You can easily replace the default homepage of Grafana in two different ways.
```

Comment:
```suggestion
You can change the default homepage of Grafana in two different ways: change Grafana preferences or change the Grafana configuration.
```

Reply:
Avoid saying something is "easy" or "simple" when giving directions. If someone has trouble, they will feel bad.

Reply:
Also, will this affect other users or just you?

Reply:
Each section has its own explanation of who will be impacted. Do you think it's enough or should I add them here?

Reply:
When I got to this sentence, the question I had was "is this change for me or will it affect all users of this instance?" If I wonder this, others will. I suggest stating that some methods change it for the individual user, other methods change it for all users of the instance, and that you note in each section what the scope of the change is.

Reply:
Reading the docs I think the information might be better structured around if you want to change the default dashboard for the organsation or for the user. I think you usually come searching for information on either rather than both. I feel the shared instruction for how to change a dashboard through preferences became a bit harder to understand than strictly necessary.

This is my attempt at a restructuring. What do you think?

---

You can change the default dashboard both on the organsation and user level. The dashboard you set as your default for the whole organisation is the one all users will see by default. Individual users can set their own default home dashboard and it will overrride whatever is set on the organisation level.

## Change Organsation default dashboard

### Setting default dashboard through preferences

### Setting default dashboard through configuration

## Change user default dashboard

only prefernces apply here

Reply:
Yes it's a good idea! I did a v3 according to this structure, I prefer this version. What do you think?